src/components/Person/index.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from "classnames";
+import * as style from "./Person.module.css";
+import {graphql, StaticQuery} from "gatsby";
+import { GatsbyImage } from "gatsby-plugin-image";
+
+const Person = ({ size= null, className = null, name, company }) => <StaticQuery
+    query={graphql`
+      query {
+        images: allFile(filter: {relativePath: {glob: "persons/*.png"}}) {
+          edges {
+            node {
+              relativePath
+                name
+                childImageSharp {
+                  gatsbyImageData(layout: CONSTRAINED)
+                }
+              }
+            }
+          }
+        }
+    `}
+    render={({ images }) => {
+        const file = images.edges.find(({node}) => node.relativePath === `persons/${name}.png`);
+
+        return <div className={cn(style.person, 'vcard', className, {
+            [style[`person_size_${size}`]]: Boolean(size),
+        })}>
+            <GatsbyImage
+                className={style.photo}
+                image={file.node.childImageSharp.gatsbyImageData}
+                alt={`${name}'s face.`}
+            />
+            <p className={style.info}>
+                <span className={cn('ktl-h4', 'n')}>{name}</span>
+                <span className={cn('ktl-text-2', 'org')}>{company}</span>
+            </p>
+        </div>
+    }}
+/>;
+
+export default Person;
```

Comment:
I would suggest not to use default import at all. https://notes.webutvikling.org/default-exports/